ci/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# This value is overridden during Concourse jobs to use a local copy of the image.
```

Comment:
Is this Dockerfile used anywhere right now?

I definitely think it would be good to put this in its own pipeline and load the baked image into ECR. It looks like there are a lot of setup tasks that run in every build.

Reply:
It is now; Docker image pipeline is defined in https://github.com/RoboJackets/concourse-pipeline-library/blob/main/pipelines/information-technology/apiary-mobile-docker-build.yml